x-pack/plugins/apm/public/components/alerting/rule_types/register_apm_rule_types.ts

```diff
@@ -55,6 +55,7 @@ export function registerApmRuleTypes(
     }),
     requiresAppContext: false,
     defaultActionMessage: errorCountMessage,
+    priority: 80,
```

Comment:
This priority will be used to order the rule type list in observability. You can check the priorities assigned to each tule in the [issue](https://github.com/elastic/kibana/issues/166136).